fastai/vision/learner.py

```diff
@@ -160,7 +160,7 @@ def _cl_int_gradcam(self, idx, heatmap_thresh:int=16, image:bool=True):
 
 ClassificationInterpretation.GradCAM =_cl_int_gradcam
 
-def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12), heatmap:bool=False, heatmap_thresh:int=16,
+def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12), heatmap:bool=False, heatmap_thresh:int=16, alpha:float=0.6, cmap:str="magma", show_text:bool=True,
```

Comment:
Can you put the extra args on the second line to make it easier to browse the code (recommended max char width is 120/140 I believe)

Reply:
Sure